appveyor.yml

```diff
@@ -0,0 +1,110 @@
+version: 1.0.{build}.{branch}
+
+environment:
+  global:
+    # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+    # /E:ON and /V:ON options are not enabled in the batch script intepreter
+    # See: http://stackoverflow.com/a/13751649/163740
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_env.cmd"
+
+  matrix:
+
+    # Pre-installed Python versions, which Appveyor may upgrade to
+    # a later point release.
+    # See: http://www.appveyor.com/docs/installed-software#python
+
+    # Python 2.6.6 is the latest Python 2.6 with a Windows installer
+    # Python 2.6.9 is the overall latest
+    # https://www.python.org/ftp/python/2.6.6/
+    - PYTHON: "C:\\Python26"
+      PYTHON_VERSION: "2.6.6"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python26-x64"
+      PYTHON_VERSION: "2.6.6"
+      PYTHON_ARCH: "64"
+
+    # Python 2.7.10 is the latest Python 2.7 with a Windows installer
+    # Python 2.7.10 is the overall latest
+    # https://www.python.org/ftp/python/2.7.10/
+    - PYTHON: "C:\\Python27"
+      PYTHON_VERSION: "2.7.10"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python27-x64"
+      PYTHON_VERSION: "2.7.10"
+      PYTHON_ARCH: "64"
+
+    # Python 3.3.5 is the latest Python 3.3 with a Windows installer
+    # Python 3.3.6 is the overall latest
+    # https://www.python.org/ftp/python/3.3.5/
+    - PYTHON: "C:\\Python33"
+      PYTHON_VERSION: "3.3.5"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python33-x64"
+      PYTHON_VERSION: "3.3.5"
+      PYTHON_ARCH: "64"
+
+    # Python 3.4.3 is the latest Python 3.4 with a Windows installer
+    # Python 3.4.3 is the overall latest
+    # https://www.python.org/ftp/python/3.4.3/
+    - PYTHON: "C:\\Python34"
+      PYTHON_VERSION: "3.4.3"
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python34-x64"
+      PYTHON_VERSION: "3.4.3"
+      PYTHON_ARCH: "64"
```

Comment:
Woo hoo! This went out before #1415 so that wasn't on my mind at the time. Will ping once updated.